README.md

```diff
@@ -1528,9 +1525,8 @@ the **settlement date**.
 
 ### Less common types of equity
 
-We’ll finish our tour of the ways equity can be granted with some other, less common kinds
-of equity. Most employees won’t run into these, but in the interest of completeness, it’s
-worth mentioning a few other flavors of equity compensation out there.
+Most employees won’t run into these, but a complete tour of the ways equity can be granted
```

Comment:
Change to:
While most employee equity compensation takes the form of stock, stock options, or RSUs, a complete tour of equity compensation must mention a few less common forms.

(Nit is imo equity can have a flavor but "ways" themselves do not." 😂 )

Reply:
...the ways are the flavors? I did not like that expression anyway so thank you for fixing 😂